Sources/DVR/Session.swift

```diff
@@ -14,6 +14,7 @@ open class Session: URLSession {
     open var recordingEnabled = true
 
     private let testBundle: Bundle
+    private let requiredHeaders: [String]
```

Comment:
This is when I figured it out. I _might_ name this something like, `headersToCheck` like you used above, as I think that's clearer.

Reply:
Yes, I can change it to be consistent.